src/components/Button/Button.tsx

```diff
@@ -1,12 +1,77 @@
-import React from "react";
-import { Button as ElButton } from "@spectral-labs/elements-react";
```

Comment:
I got an error here? It broke storybook. This line fixed it on my end. We should containerize. Looking up AWS EKS prices rn 🏃🏾‍♂️.

fix:
`import { Button as ElButton } from "../Button/index"`